### ISSUE TITLE
Can't build with liblapack

### ISSUE THREAD

**Issue by Yumin-Sun-00** [colmap/colmap#723]
**Describe the bug**
I am trying to build the Colmap.

**make -j3: messages printed in terminal**
```Scanning dependencies of target colmap_exe
[ 98%] Building CXX object src/exe/CMakeFiles/colmap_exe.dir/colmap.cc.o
make[2]: *** No rule to make target '/usr/lib/liblapack.so', needed by 'src/exe/colmap'.  Stop.
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:1184: recipe for target 'src/exe/CMakeFiles/colmap_exe.dir/all' failed
make[1]: *** [src/exe/CMakeFiles/colmap_exe.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

**liblapck path**
The path of liblapack is `/usr/lib/x86_blablabla/lapack/liblapack.so`
Could I have a hint where should I set the path correctly? I can't navigate `lapack` in the `CMakeLists.txt`

**Comment by cyh4520** [comment]
hi I have met the same question, and have you solved this problem?

**Comment by Yumin-Sun-00** [comment]
> hi I have met the same question, and have you solved this problem?

No, I tried to set search path at the top CMakeLists.txt, but it does not work for me. I only know ceres has dependency to lapack, but I build ceres successfully. So I also don't know where is the issue.

**Comment by cyh4520** [comment]
> > hi I have met the same question, and have you solved this problem?
> 
> No, I tried to set search path at the top CMakeLists.txt, but it does not work for me. I only know ceres has dependency to lapack, but I build ceres successfully. So I also don't know where is the issue.

OK thx~

**Comment by cyh4520** [comment]
hi i have just solved this problem.
The problem is that i dont have liblapck in my /usr/lib/ 
so you can cp the lib where it is (/usr/lib/x86_blablabla/lapack/liblapack.so) to /usr/lib/
and it worked for me.

**Comment by ahojnnes** [comment]
Closing as this was an environment-specific liblapack build issue from 2019. The build system has been overhauled with modern CMake/FetchContent.
